pom.xml

```diff
@@ -24,24 +24,71 @@
     <version>7</version>
   </parent>
 
-  <groupId>com.google.common.geometry</groupId>
-  <artifactId>s2-geometry-library-java</artifactId>
-  <version>HEAD-SNAPSHOT</version>
+  <groupId>com.google.geometry</groupId>
+  <artifactId>s2-geometry</artifactId>
+  <version>2.0.0</version>
   <name>S2 Geometry Library for Java</name>
   <description>
     Core geometry libraries for Java.
   </description>
+  <url>https://github.com/google/s2-geometry-library-java</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>Chris Povirk</name>
+      <email>cpovirk@google.com</email>
+      <organization>Google LLC</organization>
+      <organizationUrl>https://google.com</organizationUrl>
+    </developer>
+
+    <developer>
+      <name>David Beaumont</name>
+      <email>dbeaumont@google.com</email>
+      <organization>Google LLC</organization>
+      <organizationUrl>https://google.com</organizationUrl>
+    </developer>
+
+    <developer>
+      <name>Eric Engle</name>
+      <email>eengle@google.com</email>
+      <organization>Google LLC</organization>
+      <organizationUrl>https://google.com</organizationUrl>
+    </developer>
+
+    <developer>
+      <name>Kurt Kluever</name>
+      <email>kak@google.com</email>
+      <organization>Google LLC</organization>
+      <organizationUrl>https://google.com</organizationUrl>
+    </developer>
+  </developers>
+
+  <contributors>
+    <contributor>
+      <name>Alexei Ababilov</name>
+      <email>aababilov@google.com</email>
+      <organization>Google LLC</organization>
+      <organizationUrl>https://google.com</organizationUrl>
+    </contributor>
+  </contributors>
 
   <properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <guava.version>25.1-jre</guava.version>
```

Comment:
Git's deps tool seems to think that's the right target; is it right?

Reply:
You mean, guava 25.1-jre? Yes, it is right, it builds normally.